app/cypress/e2e/resetPassword.cy.ts

```diff
@@ -7,6 +7,7 @@ describe('Password recovery modal', () => {
   });
 
   it('gives a success message upon correct information in all steps', () => {
+    cy.log(BACKEND_URL)
```

Comment:
I know I made this line, but it should be deleted again 😆

Reply:
Done